test/DebugInfo/NonSemantic/basic-type-with-flag.spt

```diff
@@ -0,0 +1,96 @@
+;; This test checks, that SPIR-V module generated by another SPIR-V producer
+;; containing FlagUnknownPhysicalLayout flag of DebugTypeBasic can be
+;; consumed without issues
+
+; RUN: llvm-spirv %s -to-binary -o %t.spv
+; RUN: llvm-spirv -r %t.spv
+; RUN: llvm-dis < %t.bc | FileCheck %s --check-prefix=CHECK-LLVM
+
+; CHECK-LLVM: DIBasicType(name: "int", size: 32, encoding: DW_ATE_signed)
+
+119734787 65792 393230 56 0
+2 Capability Addresses
+2 Capability Linkage
+2 Capability Kernel
+2 Capability Int8
+8 Extension "SPV_KHR_non_semantic_info"
+5 ExtInstImport 1 "OpenCL.std"
+11 ExtInstImport 2 "NonSemantic.Shader.DebugInfo.100"
+3 MemoryModel 2 2
+8 String 25 "foo.cpp"
+4 String 30 "int"
+3 String 35 "A"
+3 String 36 "b"
+3 String 44 "foo"
+5 String 45 "_Z3fooP1A"
+3 String 51 "a"
+3 Source 0 0
+5 Name 7 "_Z3fooP1A"
+3 Name 8 "a"
+4 Name 9 "entry"
+4 Name 11 "a.addr"
+5 Name 16 "struct.A"
+3 Name 23 "b"
+
+15 ModuleProcessed "Debug info producer: clang version 3.1 (trunk 150996)"
+7 Decorate 7 LinkageAttributes "_Z3fooP1A" Export
+4 Decorate 11 Alignment 8
+4 TypeInt 3 32 0
+4 TypeInt 4 8 0
+4 Constant 3 21 0
+4 Constant 3 27 65536
+4 Constant 3 28 6
+4 Constant 3 31 32
+4 Constant 3 32 4
+4 Constant 3 37 1
+4 Constant 3 38 2
+4 Constant 3 41 4294967295
+4 Constant 3 46 3
+4 Constant 3 47 136
+4 Constant 3 49 16
+4 Constant 3 56 131072
```

Comment:
Nit: maybe to explicitly point out that this line is `FlagUnknownPhysicalLayout`